src/webpage/login.ts

```diff
@@ -95,7 +95,7 @@ function setDefaults(){
 		userinfos.users = {};
 	}
 	if(userinfos.accent_color === undefined){
-		userinfos.accent_color = "#242443";
+		userinfos.accent_color = "#3096f7";
```

Comment:
any reason for the change?

Reply:
Accent color has a bigger range now, in the main themes it sets the icons/reply color/mention color/outline color (the things Discord has as the blue/purple)

Reply:
Fair enough